lib/credentials/process_credentials.d.ts

```diff
@@ -0,0 +1,16 @@
+import {Credentials} from '../credentials';
+import {HTTPOptions} from '../config';
+export class ProcessCredentials extends Credentials {
+    /**
+     * Creates a new ProcessCredentials object.
+     */
+    constructor(options?: ProcessCredentialsOptions);
+}
+
+interface ProcessCredentialsOptions {
+    profile?: string
+    filename?: string
+    disableAssumeRole?: boolean
+    tokenCodeFn?: (mfaSerial: string, callback: (err?: Error, token?: string) => void) => void
+    httpOptions?: HTTPOptions
```

Comment:
I don't think we need `disableAssumeRole` or `tokenCodeFn` here, right?

Reply:
No we don't. I'll clean that and the docs up.